HHS/OPRE-OPS
# Ops 1215 agreements tabs

## What changed

Added tabs to Agreement List page: All Agreements, My Agreements

## Issue

https://github.com/HHS/OPRE-OPS/issues/1215

## How to test

1. Navigate to the Agreement List page.
2. Verify the default view lists all Agreements.
3. Add yourself as a team member to one or more Agreement.
4. Click the `My Agreements` tab and verify your Agreements are listed.

## Screenshots

![image](https://github.com/HHS/OPRE-OPS/assets/1972158/cc8ca9e3-f51e-495f-92af-98b0ef0096ed)

![image](https://github.com/HHS/OPRE-OPS/assets/1972158/1f45d21c-d7b7-4f80-9fa9-a96ce7a49c78)
